Chart.yaml

```diff
@@ -2,5 +2,5 @@ apiVersion: 'v2'
 name: 'media-servarr-base'
 description: 'Base chart for media-servarr charts'
 type: 'application'
-version: 0.9.0
+version: 0.10.0
```

Comment:
Can you remove all version upgrades from this PR?
It's currently braking linting.

As it's the base chart, every subchart will have to be updated – i'll do that immediately after this has been merged.